README.md

```diff
@@ -112,3 +121,128 @@ make
 # T2 Hardware API
 
 When you `require('tessel')` within a script which is executed on Tessel 2, this loads a library which interfaces with the Tessel 2 hardware, including pins, ports, and LEDs, just like Tessel 1 ([Tessel 1 hardware documentation](https://tessel.io/docs/hardwareAPI)). The code for Tessel 2's hardware object can be found [here](https://github.com/tessel/t2-firmware/blob/master/node/tessel.js).
+
+## Ports and pins
+
+Tessel has two ports, A and B. They are referred to as `tessel.port['A']`.
+
+Tessel's ports can be used as module ports as in Tessel 1 (e.g. `accelerometer.use(tessel.port['B'])`), or used as flexible GPIO pins (e.g. `myPin = tessel.port['A'].pins[0]`).
+
+### Modules
+
+Tessel 2's module ports can be used with [Tessel modules](//tessel.io/modules) much as in [Tessel 1](http://start.tessel.io/modules).
+
+Here is an example of using the Tessel Climate module on Tessel's port B:
+
+```js
+var tessel = require('tessel');
+var climatelib = require('climate-si7020');
+var climate = climatelib.use(tessel.port['B']);
+```
+
+### Pin mapping
+
+The module ports are not just for modules! They can also be used as flexible, simply addressable GPIO pins.
+
+The pin layout for ports A and B is as follows:
+
+| Index | Function |
+|-------|----------|
+| 0     | SCL      |
+| 1     | SDA      |
+| 2     | SCK      |
+| 3     | MISO     |
+| 4     | MOSI     |
+| 5     | TX/G1    |
+| 6     | RX/G2    |
+| 7     | G3       |
+
+G1, G2, and G3 are general purpose digital pins.
```

Comment:
All pins are general purpose digital pins, and we no longer use the G{1,2,3} numbering on T2 except for backwards compatibility